compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/input/pointer/PointerButton.skiko.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.input.pointer
+
+import androidx.compose.ui.ExperimentalComposeUiApi
+import kotlin.jvm.JvmInline
+
+@JvmInline
+@ExperimentalComposeUiApi
+value class PointerButton(val index: Int) {
+    companion object {
+        val Primary = PointerButton(0)
+        val Secondary = PointerButton(2)
```

Comment:
In Compose, Secondary button has index=1 (different from Swing)

Reply:
fixed.
But it's good for me to know where are these indexes defined for Compose?